docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -178,11 +173,16 @@ Assumptions and Limitations
 Assumptions
 +++++++++++
 
-- Presently, we are using prevalence for each stage of CKD to assign the each person in the population a CKD severity state. We are assuming (for now) that there is no transition between states. As a result, we should expect the prevalence for early stage CKD to swell as the simulation runs, since there is nowhere for these new incident cases to go. If we had remission (which is actually the term used for "progression") rates, we could use these as inter-stage incidence rates.
+- Presently, we are using prevalence for each stage of CKD to assign the each person in the population a CKD severity state. We are assuming (for now) that there is no transition between states. As a result, we should expect the prevalence for early stage CKD to swell as the simulation runs, since there is nowhere for these new incident cases to go. Transition rates (progression rates) between states are not available from the GBD model. As such, we are using evolution of risk exposure over time (changes with simulant age) to proxy for progression between CKD states - as a simulant ages, they may move to a different part of the IDF distribution, thereby landing them in a more advanced CKD state. The reason we are modeling CKD this way is because it is a condition for treatment of LDL-C, which is the intervention in this model. Thus, we need to get the prevalence at each severity (mild/moderate v. severe) correct. CKD is not a cause of interest in the current project it is being modeled in, so the severity specific prevalence is the current priority.
 
 - Simulants are in each disease state longer than they should be, compared to GBD 2017. 
 
-- This model effects other models – disability weights, excess mortality weight for each state, cause specific mortality rate
+- This model assumes there is no impact of SBP nor FPG on CKD.
+
+Limitations
++++++++++++
+
+- This model is consistent with prevalence in population. The following relationships between CKD/SBP and CKD/FPG will be modeled using correlation. The iniitial distribution will be correct, but will change over time and become inaccurate due to mitigating factors.
 
```

Comment:
Also, given that prevalence is our priority in the larger model (since CKD status impacts treatment of high LDL-C patients), we are willing to sacrifice other precision here to ensure prevalence is correct.  That the simulaiton is only 5 years long should minimize the variance in prevalence when compared to GBD.